### PR TITLE
Fix majority of linter issues including line length limit

### PR DESCRIPTION
**Pull request by paulsgrudups**
Whole code base had bunch of lines that exceeded the character limit. This should fix all of the assurances, but we should consider creating better logging functions to unify the log styles and make them bloat the code less.

Recreated selenium tests to include little bit more function calls. (tests were failing previously)
Also fixed some small recommendations by `pylint` like adding returns and converting to python 3 class definition style (no need to inherit `object`).

### CODE REVIEW

File: tests/appium_app_test.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from testui.support.testui_driver import TestUIDriver
```
[the remark this answers is not in the document]
Comment (paulsgrudups) [reply tests/appium_app_test.py:6]: Changes in file reverted to lower the scope of MR

File: tests/selenium_tests.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import pytest
```
[the remark this answers is not in the document]
Comment (paulsgrudups) [reply tests/selenium_tests.py]: Changes in file reverted to lower the scope of MR

File: testui/elements/testui_element.py [hunk 3]
```diff
@@ -4,6 +4,8 @@
 from os import path
 from typing import List
 
+from __future__ import annotations
```
Comment (ruzv) [inline testui/elements/testui_element.py:7]: this import must be the very first

**Comment by ruzv** [comment]
`.pylintrc` file is very outdated. this should be fixed. copy and paste the `.pylintrc` file from loadero-python.

**Review by ruzv — APPROVED** [review]

**Pull request merged**